events/templates/base.html

```diff
@@ -33,20 +33,28 @@
         {% endif %}
       </div>
     </nav>
-  <div class="container">
-    {% if messages %}
-      {% for message in messages %}
-        <div class="alert alert-{{ message.tags }}">
-          {{ message }}
-        </div>
-      {% endfor %}
-    {% endif %}
+    <div class="container">
+      {% if messages %}
+        {% for message in messages %}
+          <div class="alert alert-{{ message.tags }}">
+            {{ message }}
+          </div>
+        {% endfor %}
+      {% endif %}
+
+      {% block content %}
+      {% endblock content %}
+  	</div>
+    <nav class="navbar sticky-top navbar-expand-lg navbar-dark bg-dark">
```

Comment:
On my computer the About the Team bootstrap on the homepage is not completely on the bottom, whereas when I click on it it is on the bottom of the actual about page.

Reply:
This will require our own CSS classes. We'll do that later when we have a plan for our site style.

Also removed sticky-top class as it does nothing.